elastic/apm-server
# apmpackage: disallow multiple policies

## Motivation/summary

Prevent the APM integration from being added multiple times to an agent policy.

Requires #4693

## How to test these changes

1. Install apm integration
2. Add apm integration to an agent policy
3. Attempt to add another integration to the agent policy, apm integration will no longer be an option

## Related issues

Closes https://github.com/elastic/apm-server/issues/4539
